data/json/terrain.json

```diff
@@ -348,6 +365,7 @@
         "type" : "terrain",
         "id" : "t_dirtmound",
         "name": "mound of dirt",
+        "description": "An area of heaped dirt, not easily traversible but if examined more closely, they could be quite favorable for planting seeds and the like.",
```

Comment:
Should be "traversable"

Additionally, "not easily traversible" and "could be quite favorable for planting seeds" are not at odds with one another – no need for a "but."

Finally, "planting seeds and the like." Do we have things other than seeds that could be planted? 

Perhaps:

> An area of heaped dirt, not easily traversed. Could be quite favorable for planting seeds.

Reply:
Yeah, not sure how that was missed. Google seems to show both, but traversible is the depreciated form.

Wild vegetable stems, rhubarb stems, hop rhizomes, garlic cloves, raw beans, seed potato (although does have seed in the name), mushroom spores and morel mushroom spores. These are for "the like". And also allows for later changes if re-planting trees ever becomes a thing.